### PR TITLE
Add wagtail-markdown

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by jjhelmus** [comment]
LGTM

**Comment by isuruf** [comment]
@sannykr, you need to create a folder and add a `meta.yaml` inside, instead of adding a file like this.

**Comment by jakirkham** [comment]
Also there was a linter issue that was missed.

**Comment by isuruf** [comment]
This recipe was not checked at all. (linter skipped it and the CI skipped it as well)

**Comment by jakirkham** [comment]
Yeah just noting that it somehow blocked conversion. Not sure why the linter is running during conversion at all.

**Comment by jakirkham** [comment]
Put together some fixes in PR ( https://github.com/conda-forge/staged-recipes/pull/4017 ).

File: recipes/wagtail-markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+{% set version = "0.4.1" %}
+{% set sha256 = "89b24b04d874feb97adb0299b5fbc06ea29b5a835ef765b82846575a1d77f63f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchbox/wagtail-markdown/archive/0.4.1.tar.gz
```
Comment (jakirkham) [inline recipes/wagtail-markdown]: The version is hard coded here. Guessing you meant to use Jinja?

File: recipes/wagtail-markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - bleach ==1.4.2
+
+test:
+  imports:
+    - wagtailmarkdown
+
+about:
+  home: https://github.com/torchbox/wagtail-markdown
+  license: zlib
+  license_family: zlib
```
Comment (jakirkham) [inline recipes/wagtail-markdown]: This has to be changed or dropped as `license_family` can't be `zlib`.

File: recipes/wagtail-markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchbox/wagtail-markdown/archive/0.4.1.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/wagtail-markdown]: Should this be `noarch: python`?

File: recipes/wagtail-markdown [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - wagtailmarkdown
+
+about:
+  home: https://github.com/torchbox/wagtail-markdown
+  license: zlib
+  license_family: zlib
+  license_file: LICENSE
```
Comment (jakirkham) [inline recipes/wagtail-markdown]: Apparently there is no license file. Could you please raise an issue upstream and ask them to include the license file?

**Comment by jakirkham** [comment]
Have removed the recipe as it has some issue and was blocking conversion. Sorry about that @sannykr. Would encourage you to submit a fresh PR that addresses the aforementioned issues and we can get it back in. Thanks for understanding.

**Comment by sannykr** [comment]
@jakirkham : Thanks for the review. I will make a new branch and make a pull request from there.

File: recipes/wagtail-markdown (hunk 4 above)
Comment (sannykr) [reply recipes/wagtail-markdown]: Done

File: recipes/wagtail-markdown (hunk 3 above)
Comment (sannykr) [reply recipes/wagtail-markdown]: Thanks. I will make it noarch.

File: recipes/wagtail-markdown (hunk 2 above)
Comment (sannykr) [reply recipes/wagtail-markdown]: Thanks. Should I drop license as well as license_family? Although there is no license file, I found the license name in setup.py file.

File: recipes/wagtail-markdown (hunk 1 above)
Comment (sannykr) [reply recipes/wagtail-markdown]: This is not on pypi, so I used a release from github.

**Comment by jakirkham** [comment]
Thanks @sannykr. As FYI, this is now in PR ( https://github.com/conda-forge/staged-recipes/pull/4057 ).

**Pull request merged**